src/tts/magpietts/decoder.cpp

```
@@ -399,6 +403,389 @@ read_alignment_outputs(
+    const DecoderCrossKvCache& cross_kv_;
+    int text_len_ = 0;
+    int cache_len_ = 0;
+    size_t alignment_count_ = 0;
+};
+
+}  // namespace
+
+class MagpieDecoder::PersistentDecoderRuntime {
+   public:
+    PersistentDecoderRuntime(
+        const magpietts_model& model, const DecoderCrossKvCache& cross_kv, int text_len,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'PersistentDecoderRuntime' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)